components/Member.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Stack, Box, Tooltip } from "@mui/material";
+import styles from "../styles/Member.module.css";
+import { isMobile } from "react-device-detect";
+
+export interface MemberProps {
+  name: string;
+  memberInfo: string;
+  profilePic: string;
+  reverse: boolean;
+  utnevnt: string;
+  stilling: string;
+  komite: string[];
+}
+
+const Member = (member: MemberProps) => {
+  const reverse = member.reverse ? "-reverse" : "";
+  const textA = member.reverse ? "right" : "left";
```

Comment:
Could this be a more descriptive variable name?

Reply:
Or, since it is only used once (on line 48), you don't really need the alignment to be a variable